packages/core/src/Modules/SmartTraderIFrame/SmartTraderIFrame.jsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
-import { getUrlSmartTrader } from '@deriv/shared';
+import { getPlatformFromUrl } from '@deriv/shared';
 
 const SmartTraderIFrame = () => {
-    const url = getUrlSmartTrader();
-    if (!url) {
-        return null;
+    const { is_deriv_crypto, is_staging_deriv_app, is_staging_deriv_crypto } = getPlatformFromUrl();
+
+    let base_link = '';
+
+    if (is_staging_deriv_app) {
+        base_link = 'https://staging-smarttrader.deriv.com';
```

Comment:
keep the base_links in shared as constants?

Reply:
That's a good idea.